cmd/show/contributors.go

```diff
@@ -0,0 +1,190 @@
+package show
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	client "github.com/open-sauced/go-api/client"
+)
+
+// ContributorModel holds all the information related to a contributor
+type ContributorModel struct {
+	username string
+	userInfo *client.DbUser
+	userPrs  []client.DbPullRequest
+}
+
+type (
+	// BackMsg: message to signal main model that we are back to dashboard when backspace is pressed
+	BackMsg struct{}
+
+	// ContributorErrMsg: message to signal that an error occurred when fetching contributor information
+	ContributorErrMsg struct {
+		name string
+		err  error
+	}
+)
+
+// InitContributor: initializes the contributorModel
+func InitContributor(contributorName string) (ContributorModel, error) {
+	var contributorModel ContributorModel
+	contributorModel.username = contributorName
+
+	err := contributorModel.fetchUser()
+	if err != nil {
+		return contributorModel, err
+	}
+
+	return contributorModel, nil
+}
+
+func (m ContributorModel) Init() tea.Cmd { return nil }
+
+func (m ContributorModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
+	var cmd tea.Cmd
+	switch msg := msg.(type) {
+	case tea.KeyMsg:
+		switch msg.String() {
+		case "backspace":
+			return m, func() tea.Msg { return BackMsg{} }
+		case "q", "esc", "ctrl+c", "ctrl+d":
+			return m, tea.Quit
+		}
+	}
+	return m, cmd
+}
+
+func (m ContributorModel) View() string {
+	return m.drawContributorView()
+}
+
+// fetchUser: fetches all the user information (general info, and pull requests)
+func (m *ContributorModel) fetchUser() error {
+	var (
+		wg      sync.WaitGroup
+		errChan = make(chan error, 2)
+	)
+
+	wg.Add(2)
```

Comment:
I usually prefer
```
wg.Add(1)
```
above each block that launches a go routine just so it's supremely clear where the waitgroup is getting added from. More a readability thing than anything.